frontend/server/src/main/java/org/pytorch/serve/http/ManagementRequestHandler.java

```diff
@@ -329,8 +335,19 @@ private void updateModelWorkers(
                                     modelManager.scaleRequestStatus(modelName, modelVersion);
                             if (HttpResponseStatus.OK.equals(v)) {
                                 if (status) {
-                                    NettyUtils.sendJsonResponse(
-                                            ctx, new StatusResponse("Workers scaled"), v);
+                                    if (isRegistration) {
+                                        String msg =
+                                                "Model \""
+                                                        + modelName
+                                                        + "\" registered with "
+                                                        + minWorkers
+                                                        + " initial workers";
+                                        NettyUtils.sendJsonResponse(
+                                                ctx, new StatusResponse(msg), v);
+                                    } else {
+                                        NettyUtils.sendJsonResponse(
```

Comment:
create message string in the if else block and use a common statement to return the response. Add model version as well in response for model registration call.